storage.go

```diff
@@ -14,20 +14,115 @@ import (
 	. "github.com/aos-dev/go-storage/v3/types"
 )
 
+func (s *Storage) completeMultipart(ctx context.Context, o *Object, parts []*Part, opt pairStorageCompleteMultipart) (err error) {
+	if o.Mode&ModePart == 0 {
```

Comment:
https://github.com/aos-dev/go-storage/pull/573 has been implemented, so we don't need to check mode any more.